samples/system-test/subscriptions.test.js

```diff
@@ -67,7 +70,10 @@ describe('subscriptions', () => {
     const output = execSync(
       `${cmd} create-push ${topicNameOne} ${subscriptionNameTwo}`
     );
-    assert.strictEqual(output, `Subscription ${subscriptionNameTwo} created.`);
+    assert.match(
```

Comment:
I'd be tempted to just use `assert.include` (which we discussed in chat), but the match works too :+1: 

In the tests I've been fixing up I've been using `include` rather than `strictEqual` and `notInclude` rather than `strictNotEqual`.